app/src/main/java/com/example/plantsapp/presentation/ui/plantdetail/PlantDetailViewModel.kt

```diff
@@ -18,13 +19,18 @@ class PlantDetailViewModel @AssistedInject constructor(
 
     private val _plant: MutableLiveData<Plant> = MutableLiveData()
     val plant: LiveData<Plant> get() = _plant
+    private val _tasks: MutableLiveData<List<Task>> = MutableLiveData()
+    val tasks: LiveData<List<Task>> get() = _tasks
 
     private val _toNavigateBack: MutableLiveData<Event<Unit>> = MutableLiveData()
     val toNavigateBack: LiveData<Event<Unit>> get() = _toNavigateBack
 
     init {
         viewModelScope.launch {
-            _plant.value = repository.getPlantByName(Plant.Name(plantName))
+            repository.getPlantByName(Plant.Name(plantName)).run {
```

Comment:
You can write here something like: 
val (plant, tasks) = repository.getPlantByName(Plant.Name(plantName))
_plant.value = plant
 _tasks.value = tasks